testing/util/electra_state.go

```diff
@@ -143,6 +143,10 @@ func buildGenesisBeaconStateElectra(genesisTime uint64, preState state.BeaconSta
 	if err != nil {
 		return nil, err
 	}
+	pbd, err := preState.PendingBalanceDeposits()
+	if err != nil {
+		return nil, err
```

Comment:
A genesis state would not have pending balance deposits. Why do you need this change?

Reply:
might be improperly using it for my tests then I'll revert and fix